tests/phpunit/unit/test-wc-stripe-agentic-admin-capture.php

```
@@ -0,0 +1,269 @@
+		$order->set_total( 10.00 );
+		$order->set_status( OrderStatus::ON_HOLD );
+		$order->update_meta_data( '_wc_stripe_agentic_order', 'true' );
+		$order->update_meta_data( '_stripe_manual_capture', 'yes' );
+		$order->update_meta_data( '_stripe_intent_id', 'pi_test_capture' );
+		$order->update_meta_data( '_stripe_charge_captured', 'no' );
+		$order->set_payment_method( 'stripe' );
+		$order->set_transaction_id( 'pi_test_capture' );
+		$order->save();
+
+		// Mock the Stripe API response for successful capture.
+		$mock_intent = (object) [
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$mock_intent'. (undefined)

(UnusedLocalVariable)